### PR TITLE
Hardcode values replaced with reference issue

### LINKED ISSUE #8: Remove Hardcode values and consistency in all layouts
replacing all the hardcode values and reference them from values folder.

can we make the layout consistent and bring some minor changes like removing extra bottom space in recyclerview, about page etc.

@naman14 could you please add hacktoberfest lable to this issue!!

### PR DESCRIPTION
**Pull request by anandwana001**
Fixes #8 

Changes: Hardcode values replaced with reference

### CODE REVIEW

File: app/src/main/res/layout/item_status_pr.xml [hunk 1]
```diff
@@ -1,54 +1,55 @@
...
         <TextView
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
             android:id="@+id/tv_pr_title"
-            android:textSize="16sp"
-            android:paddingTop="10dp"
-            android:text="Fix issue 217 and other imrpvements" />
+            android:textSize="@dimen/text_normal"
+            android:paddingTop="@dimen/card_margin"
+            tool:text="Fix issue 217 and other imrpvements" />
```
Comment (Empty2k12) [inline app/src/main/res/layout/item_status_pr.xml]: Spelling mistake

File: app/src/main/res/layout/item_project.xml [hunk 2]
```diff
@@ -1,18 +1,20 @@
 <?xml version="1.0" encoding="utf-8"?>
 
 <android.support.v7.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
-    android:orientation="vertical" android:layout_width="match_parent"
+    xmlns:tool="http://schemas.android.com/tools"
```
Comment (Empty2k12) [inline app/src/main/res/layout/item_project.xml]: Should probably be `xmlns:tools` with a `s`.

File: app/src/main/res/layout/item_status_pr.xml (hunk 1 above)
Comment (anandwana001) [reply app/src/main/res/layout/item_status_pr.xml]: oh, I just forget to correct spelling as for this PR I only focused on removing the hardcoded values.

**Comment by Empty2k12** [comment]
This looks good, @naman14 should merge it!

**Pull request merged**